primitives/types/module.go

```diff
@@ -27,14 +27,12 @@ func GetModule(moduleIndex sc.U8, modules []Module) (Module, bool) {
 	return nil, false
 }
 
-func MustGetModule(moduleIndex sc.U8, modules []Module) Module {
+func MustGetModule(moduleIndex sc.U8, modules []Module) (Module, error) {
```

Comment:
Since this no longer returns only the module (`Must` means either return the module or panic), it can be renamed to `GetModule`